CIP-0030/README.md

```diff
@@ -164,11 +164,13 @@ type TxSignError = {
 
 In order to initiate communication from webpages to a user's Cardano wallet, the wallet must provide the following javascript API to the webpage. A shared, namespaced `cardano` object must be injected into the page if it did not exist already. Each wallet implementing this standard must then create a field in this object with a name unique to each wallet containing a `wallet` object with the following methods. The API is split into two stages to maintain the user's privacy, as the user will have to consent to `cardano.{walletName}.enable()` in order for the dApp to read any information pertaining to the user's wallet with `{walletName}` corresponding to the wallet's namespaced name of its choice.
 
-### cardano.{walletName}.enable(): Promise\<API>
+### cardano.{walletName}.enable(chainId: cbor\<chain_id> = undefined): Promise\<API>
```

Comment:
Instead of taking a position parameter, I think it's better if we make the enable function take in an object
```
{
    chainId?: cbor\<chain_id>
}
```

This makes it easier to add more options later since working with optional positional arguments gets messy

Reply:
Very true. I've corrected it. Do you see currently any more options that would impact the dapp <> wallet communication.

Reply:
Very true. I've corrected it. Do you see currently any more options that would impact the dapp <> wallet communication.